global-api/migrations/versions/88e4463fef06_locode_data_table.py

```diff
@@ -0,0 +1,50 @@
+"""locode_data_table
+
+Revision ID: 88e4463fef06
+Revises: cc8d00d4372e
+Create Date: 2024-01-17 17:44:56.668758
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import text
+
+
+# revision identifiers, used by Alembic.
+revision: str = '88e4463fef06'
+down_revision: Union[str, None] = 'cc8d00d4372e'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "city_locode",
```

Comment:
Not a great table name! How about "citywide_emissions"?

Reply:
it sounds better. I'm going to change it.